Dockerfile

```diff
@@ -14,23 +11,17 @@ RUN find modules -type f ! \( -name Gemfile -o -name "*.gemspec" -o -path "*/lib
 ###
 # shared build/settings for all child images, reuse these layers yo
 ###
-FROM ruby:3.2.2-slim-bullseye AS base
+FROM rubyimg AS base
 
 ARG userid=993
 SHELL ["/bin/bash", "-c"]
 RUN groupadd -g $userid -r vets-api && \
     useradd -u $userid -r -m -d /srv/vets-api -g vets-api vets-api
-RUN echo 'APT::Default-Release "stable";' >> /etc/apt/apt.conf.d/99defaultrelease
-RUN mv /etc/apt/sources.list /etc/apt/sources.list.d/stable.list
-RUN echo "deb http://ftp.debian.org/debian testing main contrib non-free" >> /etc/apt/sources.list.d/testing.list
-RUN echo "deb http://deb.debian.org/debian unstable main" >> /etc/apt/sources.list.d/unstable.list
+RUN echo "deb http://ftp.debian.org/debian bookworm main contrib non-free" >> /etc/apt/sources.list.d/bookworm.list
```

Comment:
Once Ruby releases a bookworm version, we can remove this line and remove `-t bookworm` from line 22. I'd expect a release in the next week.